nuclia/e2e/test_kb.py

```diff
@@ -0,0 +1,275 @@
+import pytest
+from nuclia.sdk.kb import AsyncNucliaKB, NucliaKB
+from nuclia.sdk.kbs import NucliaKBS
+from pathlib import Path
+from nucliadb_models.metadata import ResourceProcessingStatus
+from functools import wraps, partial
+from time import monotonic
+from nuclia_models.worker.tasks import TaskName, ApplyOptions, DataAugmentation
+from nuclia_models.worker.proto import (
+    ApplyTo,
+    Filter,
+    Operation,
+    Label,
+    LLMConfig,
+    LabelOperation,
+)
+from nucliadb_sdk.v2.exceptions import NotFoundError
+
+import backoff
+import asyncio
+
+ASSETS_FILE_PATH = f"{Path(__file__).parent.parent}/assets"
+
+
+@pytest.mark.asyncio
+async def wait_for(condition, max_wait=60, interval=1):
```

Comment:
This is not a test so it doesn't need to be marked as `@pytest.mark.asyncio`

Reply:
sure, search& replace when i converted from normal to async